### PR TITLE
Feature/v1-254 | Add the acquired skills to the Details page

### PR DESCRIPTION
**Pull request by dapodev**
- Frontend

1. Added Acquired Skills at Auction Details Page.
2. Reorgonized code in some places 

- Backend

1. Fixed courses and client courses technologies population

### CODE REVIEW

File: frontend/src/components/SkillInfo/SkillInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { FC } from 'react';
+
+import ProgressBar from 'components/ProgressBar/ProgressBar';
```
Comment (irina-sinitsyna) [inline frontend/src/components/SkillInfo/SkillInfo.tsx:3]:
```suggestion
import { ProgressBar } from 'components/ProgressBar';
```

File: frontend/src/pages/DetailedCourse/components/BackButton/BackButton.tsx [hunk 2]
```diff
@@ -5,9 +5,9 @@ import { ButtonLabels } from 'constants/ButtonLabels';
 import { PAGES } from 'constants/pages';
 import { PATHS } from 'constants/routes';
 import { backIconMobile } from 'icons';
-import { IBackButton } from 'types/detailedCourse';
 
 import { BackArrow, BackWrapper, StyledButton } from './styled';
+import { IBackButton } from '../../types';
```
Comment (irina-sinitsyna) [inline frontend/src/pages/DetailedCourse/components/BackButton/BackButton.tsx:10]: maybe absolute path will be better here?

File: frontend/src/pages/DetailedCourse/types.ts [hunk 3]
```diff
@@ -1,3 +1,80 @@
+import { NavigateFunction } from 'react-router';
+
+import { CourseStatus } from 'enums/course';
+
+import { PureClientCourse } from 'types/clientCourse';
```
Comment (irina-sinitsyna) [inline frontend/src/pages/DetailedCourse/types.ts:5]:
```suggestion
import { CourseStatus } from 'enums/course';
import { PureClientCourse } from 'types/clientCourse';
```

File: frontend/src/components/SkillInfo/SkillInfo.tsx (hunk 1 above)
Comment (ostapdans99) [reply frontend/src/components/SkillInfo/SkillInfo.tsx]: done

File: frontend/src/pages/DetailedCourse/components/BackButton/BackButton.tsx (hunk 2 above)
Comment (ostapdans99) [reply frontend/src/pages/DetailedCourse/components/BackButton/BackButton.tsx]: done

File: frontend/src/pages/DetailedCourse/types.ts (hunk 3 above)
Comment (ostapdans99) [reply frontend/src/pages/DetailedCourse/types.ts]: done

**Review by irina-sinitsyna — APPROVED** [review]

**Review by irina-sinitsyna — DISMISSED** [review]

**Review by irina-sinitsyna — APPROVED** [review]

**Pull request merged**